Amplify/Categories/Storage/StorageCategory+ClientBehavior.swift

```diff
@@ -89,4 +89,10 @@ extension StorageCategory: StorageCategoryBehavior {
                 options: StorageGetURLOperation.Request.Options?) async throws -> URL {
         try await plugin.getURL(key: key, options: options)
     }
+
+    @discardableResult
+    public func remove(key: String,
+                       options: StorageRemoveRequest.Options? = nil) async throws -> String {
+        try await plugin.remove(key: key, options: options)
```

Comment:
looks like this API is correct in having a default value of `nil` for the options parameter, if so then we should add it for getURL's options paramter as well